templates/embedded/index.html

```diff
@@ -494,34 +494,31 @@ <h3>All kinds of containers</h3>
     <div class="col-6 p-card">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/944b8760-snapcraft-logo-bird.svg",
-          alt="",
-          height="67",
-          width="64",
+          url="https://assets.ubuntu.com/v1/18f09ca7-snaps-logo-34.svg",
+          alt="Canonical snaps",
+          width="172",
+          height="33",
           hi_def=True,
           loading="lazy",
-          attrs={"class": "p-card__thumbnail", "style": "max-height: 4rem;"}
+          attrs={"class": "p-card__thumbnail", "style": "border-radius: 0"}
         ) | safe
       }}
       <hr class="u-sv1">
-      <h3>Snaps</h3>
       <p>App packages for desktop, cloud and IoT that are easy to install, secure, cross‐platform and dependency‐free. Snaps are discoverable and installable from the Snap Store, the app store for Linux with an audience of millions.</p>
-      <p><a href="https://snapcraft.io/first-snap">Build a snap</a></p>
+      <p class="p-card__content"><a href="https://snapcraft.io/first-snap">Build a snap</a></p>
```

Comment:
Is there any need for the `p-card__content class`? The links in other cards don't have it. I don't know if there's any downside per se but I was just wondering. I'm going to approve anyway, everything looks good.

Reply:
It should be wrapped around the content maybe. It reduces a bit the spacing, but in this case I don't think it makes a big difference. I can remove it